### PR TITLE
PR templates

### PR DESCRIPTION
**Pull request by neelgala**
- adding a pull request template
  - removed riscv-test-stats directory and migrated those to a gdrive folder [here](https://drive.google.com/drive/folders/1KBRy6OgxnOPTDgyfJDj0gcMi5VdMLtVo?usp=share_link)
  - updated contribution guide on steps to done to upload test stats for PRs

### CODE REVIEW

**Review by InspireSemi — APPROVED** [review]
Looks good

File: .github/PULL_REQUEST_TEMPLATE/add_modify_tests.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- [ ] SAIL
+- [ ] Spike
+- [ ] Other (please specify below)
+
+## Mandatory Checklist:
+
+  - [ ] All tests are compliant with the test-format spec present in this repo ?
+  - [ ] Ran the new tests on RISCOF with SAIL/Spike as reference model successfully ?
+  - [ ] Ran the new tests on RISCOF in [coverage mode](https://riscof.readthedocs.io/en/stable/commands.html#coverage)
+  - [ ] Link to Google-Drive folder containing the new coverage reports ([See this]() for more info)
```
Comment (allenjbaum) [inline .github/PULL_REQUEST_TEMPLATE/add_modify_tests.md]: needs a real link, yes?

File: .github/PULL_REQUEST_TEMPLATE/add_modify_tests.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  - [ ] Link to PR in RISCV-ISAC from which the reports were generated.
+  - [ ] Changelog entry created with a minor patch
+
+## Optional Checklist:
+
+  - [ ] Were the tests generated using RISCV-CTG ? If so please provide the link to the PR on RISCV-CTG
+  - [ ] Were the tests hand-written/modified ?
+  - [ ] Have you run these on any hard DUT model ? Please specify name and provide link if possible
+  - [ ] If you have modified arch\_test.h Please provide a detailed description of the changes in
```
Comment (allenjbaum) [inline .github/PULL_REQUEST_TEMPLATE/add_modify_tests.md]: Backslash shouldn't bee there

File: .github/PULL_REQUEST_TEMPLATE/add_modify_tests.md (hunk 1 above)
Comment (neelgala) [reply .github/PULL_REQUEST_TEMPLATE/add_modify_tests.md]: this file is not longer on the final PR - were you reviewing a commit instead ?

**Comment by allenjbaum** [comment]
Ah, yes I was; I am guessing a later commit fixed this?

**Review by pawks — APPROVED** [review]

**Pull request merged**